Scoop/NetworkManager.swift

```diff
@@ -180,23 +180,24 @@ class NetworkManager {
             let jsonDecoder = JSONDecoder()
             jsonDecoder.dateDecodingStrategy = .iso8601
             jsonDecoder.keyDecodingStrategy = .convertFromSnakeCase
-            if let ridesResponse = try? jsonDecoder.decode(Ride.self, from: data) {
-                completion(ridesResponse)
-            } else {
-                print("Failed to decode getSpecific Ride from JSON")
+            do {
+                let ride = try jsonDecoder.decode(Ride.self, from: data)
+                completion(.success(ride))
+            } catch {
+                completion(.failure(error))
             }
         case .failure(let error):
             print(error.localizedDescription)
             }
```

Comment:
cant tell if it's a github viewing issue but the indentation looks a bit weird with the curly brackets

Reply:
haha it's me, the indentation was a bit funky there